packages/dropdown/src/react/item.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import { css } from 'glamor'
+import ActionMenu from '@pluralsight/ps-design-system-actionmenu'
+
+import stylesheet from '../css/index.js'
+const styles = () => css(stylesheet[`.psds-dropdown__item__icon`])
+
+export const Item = ({ value, icon, children, ...rest }) => {
+  return (
+    <ActionMenu.Item tag="button" value={value} {...rest}>
+      {icon && <span {...styles()}>{icon}</span>}
+      <ActionMenu.Ellipsis>{children}</ActionMenu.Ellipsis>
```

Comment:
How do you both feel about doing ellipsis this way in order to handle various menu item layouts?